connect/mirror/src/test/resources/log4j2.properties

```diff
@@ -14,24 +14,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ##
-log4j.rootLogger=INFO, stdout
+name=ConnectMirrorTestConfig
```

Comment:
I don't understand the changes in the file. We move a log4j.properties in `connect/runtime` to `connect/mirror`? Why?

Reply:
Hmm... I think it's just an issue with GitHub detection. I modified these two files in place. 🤔